nulib/authoritex
# Update livebook to use Kino.Input functions for the search example

Livebook has offloaded input functionality to [kino](https://github.com/livebook-dev/kino), and this PR updates the Authoritex livebook document to use the `Kino.Input.text/1` and `Kino.Input.select/2` functions.

Steps to test:
- Spin up a livebook server on your dev machine (instructions here: https://livebook.dev/#install)
- Go to the livebook landing page and click the `Import` button:
![livebook_import](https://user-images.githubusercontent.com/1395707/160485110-db7ad36c-dbe4-4865-9cde-f182274b27a4.jpg)
- Enter the following URL into the text input field: `https://raw.githubusercontent.com/nulib/authoritex/livebook-update/LIVEBOOK.livemd`
![import_notebook](https://user-images.githubusercontent.com/1395707/160485258-23fd7eb6-6089-46ca-ad75-ec78aac2b4dd.png)
- The notebook should load without any pop-up warnings, and all the functionality should work as expected
